### PR TITLE
Update to nodejs 10

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem
Nodejs 8 outdated

## Solution
JupyterLab version 2.1 [Support Node.js 10+ (#8112, #8083)](https://jupyterlab.readthedocs.io/en/stable/getting_started/changelog.html)

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** : 
- [ ] **Approval (LGTM)** : 
- **CI succeeded** : 
[Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/jupyterlab_ynh%20PR-NUM-/)  
*Please replace '-NUM-' in this link by the PR number.*  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

File: scripts/install [hunk 1]
```diff
@@ -80,7 +80,7 @@ ynh_script_progression --message="Installing dependencies..." --weight=23
 
 ynh_install_app_dependencies $pkg_dependencies
 
-ynh_exec_warn_less ynh_install_nodejs --nodejs_version=8
+ynh_exec_warn_less ynh_install_nodejs $nodejs_version
```
Comment (kay0u) [inline scripts/install:83]:
```suggestion
ynh_exec_warn_less ynh_install_nodejs --nodejs_version=$nodejs_version
```

File: scripts/restore [hunk 2]
```diff
@@ -75,7 +75,7 @@ ynh_script_progression --message="Reinstalling dependencies..." --weight=75
 # Define and install dependencies
 ynh_install_app_dependencies $pkg_dependencies
 
-ynh_exec_warn_less ynh_install_nodejs --nodejs_version=8
+ynh_exec_warn_less ynh_install_nodejs $nodejs_version
```
Comment (kay0u) [inline scripts/restore:78]:
```suggestion
ynh_exec_warn_less ynh_install_nodejs --nodejs_version=$nodejs_version
```

File: scripts/upgrade [hunk 3]
```diff
@@ -95,7 +95,7 @@ ynh_script_progression --message="Upgrading dependencies..." --weight=83
 
 ynh_install_app_dependencies $pkg_dependencies
 
-ynh_exec_warn_less ynh_install_nodejs --nodejs_version=8
+ynh_exec_warn_less ynh_install_nodejs $nodejs_version
```
Comment (kay0u) [inline scripts/upgrade:98]:
```suggestion
ynh_exec_warn_less ynh_install_nodejs --nodejs_version=$nodejs_version
```

Comment (ericgaspar) [reply scripts/upgrade:98]: Yes of course 🙄

**Pull request merged**